Generate.ps1

```diff
@@ -5,7 +5,7 @@ $AZURE_ARGUMENTS = "--version=$AUTOREST_CORE_VERSION --java --use=. --output-fol
 $PROTOCOL_ARGUMENTS = "--version=$AUTOREST_CORE_VERSION --java --use=. --output-folder=protocol-tests --data-plane --generate-samples"
 $PROTOCOL_RESILIENCE_ARGUMENTS = "--version=$AUTOREST_CORE_VERSION --java --use=. --data-plane"
 $SWAGGER_PATH = "node_modules/@microsoft.azure/autorest.testserver/swagger"
-$PARALLELIZATION = 3
+$PARALLELIZATION = 1
```

Comment:
Can we make this a command line arg in CI to set a higher value there and keep the parallelization to 1 in local dev boxes?